src/components/Popup.js

```diff
@@ -1,11 +1,20 @@
 export default class Popup {
   constructor({ popupSelector }) {
     this._popupElement = document.querySelector(popupSelector);
+    this._submitButton = this._popupElement.querySelector(
+      ".modal__save-button"
+    );
+    this._popupSelector = popupSelector;
   }
 
   open() {
     this._popupElement.classList.add("modal_opened");
     document.addEventListener("keydown", this._handleEscClose);
+    if (this._popupSelector == "#card-add-modal") {
```

Comment:
Needs correction: Please don't change the button text here. The `Popup` class should not know anything about the submit buttons and the specific popups.